app/src/main/java/net/squanchy/search/view/SearchAdapter.java

```diff
@@ -32,18 +32,18 @@
     }
 
     private final ImageLoader imageLoader;
-    private final Context context;
+    private final Activity activity;
 
     @Nullable
     private SearchRecyclerView.OnSearchResultClickListener listener;
 
     private SearchResults searchResults = SearchResults.create(Collections.emptyList(), Collections.emptyList());
     private ItemsAdapter itemsAdapter = new ItemsAdapter(searchResults);
 
-    SearchAdapter(Context context) {
-        this.context = context;
+    SearchAdapter(Activity activity) {
```

Comment:
nitpick (not for this PR), `SearchAdapter` sounds like a dependency, probably it should be provided an `ImageLoader` and `Activity` and be provided itself in a `SearchModule`